### PR TITLE
feat: add uuid tag

### LINKED ISSUE #823: A component to display uuid
UUID is too long in some case, so we'd better has a wey to display the id, some consideration, any suggestion is welcome:

1. Omit part of the uuid, say, `b1e2...55cb`
2. Be able to check the full id by hovering.
3. Styleless, so we can re-use it in all kinds of environment, maybe use `span` tag is good idea, because we dont' want to break the line as shown below:

Image

### PR DESCRIPTION
**Pull request by RubenSmn**
# Description
Add plain box component and sx for styling
Fixes #823

## Type of change

[comment]: # 'Please delete options that are not relevant.'

- [x] New feature (non-breaking change which adds functionality)

## Screenshots
image

# Checklist:

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/components/common/UUIDTag.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Box } from '@mui/material';
+
+function UUIDTag({ uuid, sx }) {
+  const formattedId = `${uuid.slice(0, 4)}...${uuid.slice(
+    uuid.length - 4,
+    uuid.length,
+  )}`;
+
+  return <Box sx={sx}>Token #{formattedId}</Box>;
```
Comment (dadiorchen) [inline src/components/common/UUIDTag.js:9]: @RubenSmn great you are working on this, I have some suggestion:

1. `Token #` shouldn't be the knowledge of this component, so we can reuse it somewhere, say: the tree uuid (the tree id would be in form of uuid in the future) 

2. I need a tooltip to display the entire uuid by hovering the mouse. 
3. I need the tooltip to be interactable, so I can cope the id ( for example, a button/link on the tooltip saying: `copy`
4. Use `span` as the tag of this component, so we can embedded it in to the context without breaking the line and we can inherite the style, ( I remember we can indicate the tag for box and so on, like: `<Box component='span' />`, ideally I hope we can simply bring in the function (shorten the id) and do not invade the style.

**Comment by RubenSmn** [comment]
Change the component to be a `span`
Add basic `tooltip` with extra font size for better UX

**Comment by dadiorchen** [comment]
@RubenSmn will you add the `copy` function for this component? Or I merge it first?

**Comment by RubenSmn** [comment]
@dadiorchen I will add the copy aswell

**Comment by RubenSmn** [comment]
[Kooha-2022-09-03-10:46:19.webm](https://user-images.githubusercontent.com/31519867/188263311-52772861-4587-46bc-bf22-e841a85d8b44.webm)

**Comment by dadiorchen** [comment]
THis is really nice

**Pull request merged**